PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterCharacterScripts/Sound.rbxmx

```diff
@@ -83,7 +85,15 @@ local Sounds = {}
 do
 	local Figure = script.Parent.Parent
 	Head = Figure:WaitForChild("Head")
-	Humanoid = Figure:WaitForChild("Humanoid")
```

Comment:
doesn't this still bode the same for head? what if someone renames head? or if the object with the name "Head" is no longer the head? Do we need another WaitForChild? At the least shouldn't we wait for both of these in the same way?